docs/layer_util.js

```diff
@@ -255,6 +255,17 @@ function addLayer(layerName, index, map) {
   }
 }
 
+/**
+ * Given a layer name, turns it into an ImageCollection. Masks images with
+ * themselves to avoid displaying black pixels (which are usually just points
+ * that weren't captured by the imagery).
+ * @param {string} layerName Name of ImageCollection
+ * @return {ee.ImageCollection}
+ */
+function processImageCollection(layerName) {
+  return ee.ImageCollection(layerName).map((image) => image.selfMask());
```

Comment:
Sorry I wrote this earlier today and failed to hit submit review!

Reply:
Good question! I guess it's possible, but I haven't actually seen it, because we're not really using images much. This does make black "transparent", so if there are real black pixels, it could be a confusing experience (although I think true black basically never happens except for missing data). I'd prefer to leave normal images as they are until we see a need?

Reply:
sgtm